scripts/build_renewable_profiles.py

```diff
@@ -60,7 +60,7 @@
     **Source:** `GEBCO <https://www.gebco.net/data_and_products/images/gebco_2019_grid_image.jpg>`_
 
 - ``resources/natura.tiff``: confer :ref:`natura`
-- ``resources/country_shapes.geojson``: confer :ref:`shapes`
+- ``resources/ohshore.tiff``: confer :ref:`onshore`
```

Comment:
Is this right?

country shapes is still in the Snakefile and I haven't seen a `onshore.tiff` yet (besides the typo)

Reply:
Good catch, that is an artifact from previous attempts to compute a distance field for onshore regions.